spec/validations/manifest_spec.rb

```diff
@@ -864,4 +857,117 @@ def create_package(parameter_hash)
       end
     end
   end
+
+  context 'scope parameter validations' do
+    context 'when validate_scopes_for_secure_parameter is false' do
+      it 'should have an invalid_keys error when scopes key is present' do
+        @manifest_hash = {
+          'parameters' => [
+            {
+              'name' => 'api_token',
+              'type' => 'text',
+              'secure' => true,
+              'scopes' => ['header']
+            }
+          ]
+        }
+        package = create_package(@manifest_hash)
+        errors = ZendeskAppsSupport::Validations::Manifest.call(package, validate_scopes_for_secure_parameter: false)
+        expect(errors.map(&:to_s)).to include('parameters[name="api_token"] contains invalid keys: ["scopes"].')
```

Comment:
Like the "invalid values" error message, how about removing the square brackets here as well?

```suggestion
        expect(errors.map(&:to_s)).to include('parameters[name="api_token"] contains invalid keys: scopes.')
```